src/core/store/modules/vaults/vaults.actions.ts

```diff
@@ -204,6 +205,36 @@ const approveZapOut = createAsyncThunk<void, { vaultAddress: string }, ThunkAPI>
   }
 );
 
+const signZapOut = createAsyncThunk<{ signature: string }, { vaultAddress: string }, ThunkAPI>(
+  'vaults/signZapOut',
+  async ({ vaultAddress }, { getState, extra }) => {
+    const { network, wallet } = getState();
+    const { vaultService } = extra.services;
+    const { CONTRACT_ADDRESSES } = extra.config;
+
+    // NOTE: this values are hardcoded on zappers zapOut contract
+    const amount = '79228162514260000000000000000'; // https://etherscan.io/address/0xd6b88257e91e4E4D4E990B3A858c849EF2DFdE8c#code#F8#L83
+    const deadline = '0xf000000000000000000000000000000000000000000000000000000000000000'; // https://etherscan.io/address/0xd6b88257e91e4E4D4E990B3A858c849EF2DFdE8c#code#F8#L80
```

Comment:
Did not add to config/constants since will only be used here and will have no context under other file. Not sure why zapper hardcode this specific values, will ask them to provide information.